operator/cfg/config.go

```diff
@@ -32,98 +32,56 @@ const (
+	Config = &Configuration{}
+)
+
 // Configuration holds a strongly-typed tree of the configuration
 type Configuration struct {
-	MountPath                        string `koanf:"datapath"`
-	BackupAnnotation                 string `koanf:"annotation"`
-	BackupCommandAnnotation          string `koanf:"backupcommandannotation"`
-	FileExtensionAnnotation          string `koanf:"fileextensionannotation"`
-	ServiceAccount                   string `koanf:"podexecaccountname"`
-	BackupCheckSchedule              string `koanf:"checkschedule"`
-	GlobalAccessKey                  string `koanf:"globalaccesskeyid"`
-	GlobalKeepJobs                   int    `koanf:"globalkeepjobs"`
-	GlobalFailedJobsHistoryLimit     int    `koanf:"global-failed-jobs-history-limit"`
-	GlobalSuccessfulJobsHistoryLimit int    `koanf:"global-successful-jobs-history-limit"`
-	GlobalRepoPassword               string `koanf:"globalrepopassword"`
-	GlobalRestoreS3AccessKey         string `koanf:"globalrestores3accesskeyid"`
-	GlobalRestoreS3Bucket            string `koanf:"globalrestores3bucket"`
-	GlobalRestoreS3Endpoint          string `koanf:"globalrestores3endpoint"`
-	GlobalRestoreS3SecretAccessKey   string `koanf:"globalrestores3secretaccesskey"`
-	GlobalS3Bucket                   string `koanf:"globals3bucket"`
-	GlobalS3Endpoint                 string `koanf:"globals3endpoint"`
-	GlobalSecretAccessKey            string `koanf:"globalsecretaccesskey"`
-	GlobalStatsURL                   string `koanf:"globalstatsurl"`
-	GlobalConcurrentArchiveJobsLimit int    `koanf:"global-concurrent-archive-jobs-limit"`
-	GlobalConcurrentBackupJobsLimit  int    `koanf:"global-concurrent-backup-jobs-limit"`
-	GlobalConcurrentCheckJobsLimit   int    `koanf:"global-concurrent-check-jobs-limit"`
-	GlobalConcurrentPruneJobsLimit   int    `koanf:"global-concurrent-prune-jobs-limit"`
-	GlobalConcurrentRestoreJobsLimit int    `koanf:"global-concurrent-restore-jobs-limit"`
-	GlobalCPUResourceRequest         string `koanf:"global-cpu-request"`
-	GlobalCPUResourceLimit           string `koanf:"global-cpu-limit"`
-	GlobalMemoryResourceRequest      string `koanf:"global-memory-request"`
-	GlobalMemoryResourceLimit        string `koanf:"global-memory-limit"`
-	BackupImage                      string `koanf:"image"`
-	MetricsBindAddress               string `koanf:"metrics-bindaddress"`
-	PodExecRoleName                  string `koanf:"podexecrolename"`
-	PodFilter                        string `koanf:"podfilter"`
-	PromURL                          string `koanf:"promurl"`
-	RestartPolicy                    string `koanf:"restartpolicy"`
+	MountPath                        string
+	BackupAnnotation                 string
+	BackupCommandAnnotation          string
+	FileExtensionAnnotation          string
+	ServiceAccount                   string
+	BackupCheckSchedule              string
+	GlobalAccessKey                  string
+	GlobalKeepJobs                   *int
```

Comment:
Did you miss this one regarding pointers?

Reply:
🤦